nuxeo-features/nuxeo-thumbnail/src/test/java/org/nuxeo/ecm/platform/thumbnail/test/TestRecomputeThumbnails.java

```diff
@@ -53,11 +53,6 @@
 @RunWith(FeaturesRunner.class)
 @Features(AutomationFeature.class)
 @Deploy("org.nuxeo.ecm.platform.convert")
-@Deploy("org.nuxeo.ecm.automation.server")
-@Deploy("org.nuxeo.ecm.automation.io")
-@Deploy("org.nuxeo.ecm.webengine.core")
-@Deploy("org.nuxeo.ecm.platform.forms.layout.export")
-@Deploy("org.nuxeo.ecm.platform.rendition.core")
```

Comment:
Why?

Reply:
The test worked without it. This was only tested after switching to BAF so I assumed it was because I dropped the work approach